Backend/API/Controllers/AppConrtoller.cs

```diff
@@ -128,5 +128,18 @@ public IActionResult GetTask(int id)
             var model = _mapper.Map<IList<Models.Task.TaskModel>>(tasks);
```

Comment:
```suggestion
            // Uses automapper to map the result as a list of Models.Taks.TaskModel 
            // (typecast to ensure the correct datatype instead of Data Transfer Object)
            var model = _mapper.Map<IList<Models.Task.TaskModel>>(tasks);
```